lib/modules/manager/circleci/extract.ts

```diff
@@ -5,19 +5,22 @@ import { OrbDatasource } from '../../datasource/orb';
 import * as npmVersioning from '../../versioning/npm';
 import { getDep } from '../dockerfile/extract';
 import type { PackageDependency, PackageFileContent } from '../types';
-import { CircleCiFile, type CircleCiJob } from './schema';
+import {
+  CircleCiFile,
+  type CircleCiFile as CircleCiFileType,
```

Comment:
don't rename

Reply:
Why habe you resolved this?

Reply:
ah sorry, I assumed the comment was about the non-duplicate imports. I wasn't able to import `CircleCiFile` and `type CircleCiFile` without the rename, but that might be my lack of typescript knowledge. can you recommend what I should do?

Reply:
simply remove the second import, typescript can handle it

Reply:
🤦  done, thank you